DBSizingQueries/Parse-RubrikSizing.ps1

```diff
@@ -7,11 +7,11 @@ $data = Get-Content $csvfile
 if ($data[0].Substring(1,10) -ne "ServerName")
 {
     $Header ="ServerName","SQLVersion","name","recovery_model_desc","SevenDayLogBackupMB","AverageFullMB","AverageFullTimeSec","AverageLogTimeSec","DBTotalSizeMB","AverageLogBackupInterval","ChangeCapture","Compression","FILESTREAM","InMemoryOLTP","Partitioning","TransparentDataEncryption"
-    $rawdata = Get-Content $csvfile | ConvertFrom-Csv -Delimiter $delimiter -Header $Header
+    $rawdata = $(Get-Content $csvfile) -replace ",","." | ConvertFrom-Csv -Delimiter $delimiter -Header $Header
```

Comment:
Please clarify this. Is this to handle European formatting (1.000 vs. 1,000)? This would break the file if they actually ended giving us a comma delimited file.